packages/lib/CalendarManager.ts

```diff
@@ -54,13 +54,14 @@ export const getConnectedCalendars = async (
     calendarCredentials.map(async (item) => {
       try {
         const { integration, credential } = item;
+        const safeToSendIntegration = cleanIntegrationKeys(integration);
```

Comment:
Ensuring that we use existing `cleanIntegrationKeys` so that `credential`/`credentials` aren't there